### ISSUE TITLE
Login issue - Null Redirection URI

### ISSUE THREAD

**Issue by jlev** [FIRST-Tech-Challenge/fmltc#238]
Not sure if this is the right repository, but I tried to log into ftc-ml.firstinspires.org for the first time, and got an error that `Redirection URI must not be null.` I had a valid login session, so clearing the url and reloading the page worked fine.

Screenshot with traceback attached:
<img width="1565" alt="Screen Shot 2021-12-06 at 12 13 58 PM" src="https://user-images.githubusercontent.com/294964/144916073-635a082b-a4ed-4945-be61-6f2b5ce3858b.png">

**Comment by texasdiaz** [comment]
It appears you clicked on the "Hello, Josh Levinger" and it took you to the ftc-scoring accounts page (since the Dashboard SSO is managed by ftc-scoring). Can you confirm? Do you have a team affiliation set in your FIRST Dashboard?

**Comment by chaosbuster** [comment]
Saw the same Java error after accepting FML-TC agreement.  Firefox browser.  Was logged into FIRST Inspires Dashboard before submitting.

Reloaded page and it went to page to select which of my teams.  Was able to select a team and get to the tool.  

Team members are not able to access it though.  They are accepted team members with consent & release on my Dashboard in firstinspires.org

**Comment by cmacfarl** [comment]
You want to take a look at this post regarding granting team members
access.

https://community.ftclive.org/t/granting-students-team-members-access/45

**Comment by chaosbuster** [comment]
Yes, we found this also in the FML Toolchain Manual. Thanks for sharing!  We are following the steps mentioned, but the student team member is still having problems when I add the student email within first inspired.org.  I think his dad registered and added him so...maybe the student doesn't have an account on first inspires org.  Wondering if we should just add the parent's email as a "Team member" within the FTC Scoring?

**Comment by cmacfarl** [comment]
It is possible for a parent to create a dashboard account for a child but not provide that child with an email address.  In this scenario, the child doesn't have a full account on the FIRST Inspires dashboard and hence can not log into the system.

Could this perhaps be what is going on with your student?

Note for the wall of red text above, that's a known transient error within ftc-scoring.  Refresh and it should go away.  That however is unrelated to your student not being able to log in.

**Comment by cmacfarl** [comment]
Using the name in the screen shot above, I was able to determine what team you are associated with.  ftc-scoring.firstinspires.org, for that team, only has the two lead mentors (Team Administrators) in the list of users.

I presume you tried to add this team member to that list per the instructions given [here](https://community.ftclive.org/t/granting-students-team-members-access/45).

If that didn't work, confirm that the student has a full dashboard account, and confirm that you used the correct email.  If that doesn't work, try another student.  If none of that is working for you, please send email to ftctech@firstinspires.org with some screenshots and full details on the team and students you are trying to get access to system for.

**Comment by chaosbuster** [comment]
Thank you, Chris, for your comments, especially during the holidays.  

I created a [separate Issue #245 in GitHub for this discussion](https://github.com/FIRST-Tech-Challenge/fmltc/issues/245) to keep this original bug to topic.  ;-)
